BrainPortal/app/views/nh_invitations/_message_details.html.erb

```diff
@@ -0,0 +1,52 @@
+
```

Comment:
This file seems like aalso a almost complete duplication of `messages/_message_details.html.erb`, can we instead use THAT and not duplicate so much code?

Reply:
Surely, it is easy to adjust that file. That what I done in an early version of the feature. I opted to allow Neurohub have it own template that can evolve independtly, but totally can drop that

Reply:
OK, I agree with this sentiment. Let's have a separate file for neurohub, like you did.

Reply:
You convinced me.